tests/parametric/test_config_consistency.py

```diff
@@ -311,7 +311,7 @@ def test_setting_trace_rate_limit(self, library_env, test_agent, test_library):
 
 
 @scenarios.parametric
-@features.tracing_configuration_consistency
+@features.dd_tags
```

Comment:
This stands out compared to the other feature names, but I'm not sure `trace_tags` accurately covers this.